app/src/main/java/com/gh4a/db/BookmarksProvider.java

```diff
@@ -83,6 +102,14 @@ public static boolean hasBookmarked(Context context, String url) {
         return hasBookmarked;
     }
 
+    public static void reorderBookmark(Context context, long id, int orderId) {
+        ContentValues cv = new ContentValues();
+        cv.put(Columns.ORDER_ID, orderId);
+        context.getContentResolver().update(Columns.CONTENT_URI, cv,
```

Comment:
Can use ContentUris.withAppendedId(Columns.CONTENT_URI, id) here instead of doing the selection below.